test/integration/targets/azure_rm_aci/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+- name: Create sample container group
+  azure_rm_aci:
+    resource_group: "{{ resource_group }}"
+    name: mynewcontainergroup
```

Comment:
what is the name is already there? maybe name: "aci{{ resource_group | hash('md5') | truncate(7, True, '') }}" better?

Reply:
that's in ansible's test framework

Reply:
ah, yes, i misunderstood.
well, first of all container instance can be created only in an empty resource group, otherwise will fail.
so it's not a problem here.
I don't really know mechanics of ansible test framework, but seems like every subsequent test run gives me empty resource group